repair/row_level.cc

```diff
@@ -3007,6 +3007,15 @@ class row_level_repair {
         if (!_shard_task.hints_batchlog_flushed()) {
             co_return;
         }
+
+        // The tablet repair time for tombstone gc will be updated when the
+        // system.tablet.repair_time is updated.
+        if (_is_tablet && _shard_task.sched_by_scheduler) {
```

Comment:
The RPC is is internal to repair. It has nothing to do with the raft coordinator. Repair knows exactly how it has repaired with. The sched_by_scheduler flag guarantees the coordinator uses the new logic to update the repair time. No need to introduce a feature flag. Note this will release  with the repair scheduler code together.

Reply:
If there is no mixed cluster, then there is no problem indeed.